reposilite-backend/src/main/kotlin/com/reposilite/console/application/ConsolePlugin.kt

```diff
@@ -91,6 +112,10 @@ internal class ConsolePlugin : ReposilitePlugin() {
 
         event { _: ReposiliteDisposeEvent ->
             consoleFacade.commandExecutor.stop()
+            client.users.forEach {
+                it.key.close()
+            }
+            watcher.cancel(false)
```

Comment:
Related to what I said before - it'd be nice mange the state in the service class.

And watcher should be probably disabled before closing the sse clients :thinking: